tests/syntax/test_specifiers.py

```diff
@@ -505,12 +505,44 @@ def test_visible():
         "             with visibleDistance 10, with viewAngle 90 deg\n"
         "ego = new Object visible"
     )
+    radius = math.sqrt(3 * 0.5**2)
```

Comment:
`math.hypot(0.5, 0.5, 0.5)` would be clearer here I think.

Reply:
Sure, I swapped all occurrences in the file.